client/my-sites/plan-features-2023-grid/types.ts

```diff
@@ -8,8 +8,9 @@ export type TransformedFeatureObject = FeatureObject & {
 
 export interface PlanActionOverrides {
 	loggedInFreePlan?: {
-		callback: () => void;
-		text: TranslateResult;
+		callback?: () => void;
+		text?: TranslateResult;
+		status?: 'blocked' | 'enabled';
```

Comment:
Nit: it looks like the only value in use is `'blocked'`, which means it's likely can be just `isBusy` of boolean value, but I'm fine with leaving it this way :)

Reply:
I actually was struggling to find a proper term but I think I just got it. How about, 'disabled' | 'enabled' | 'busy' ?